social_django/compat.py

```diff
@@ -0,0 +1,23 @@
+# coding=utf-8
+import six
+import django
+from django.db import models
+
+if django.VERSION >= (2, 0):
+    from django.urls import reverse
+else:
+    from django.core.urlresolvers import reverse
+
+assert reverse
```

Comment:
Please remove this `assert`, this mechanism is not used in the rest of the codebase.

Reply:
The following code triggers IDE warnings:
```py
try:
    from django.utils.encoding import smart_unicode as smart_text
    smart_text  # placate pyflakes
except ImportError:
```
assert is better in this respect. Or you insist?